packages/flutter_driver/lib/src/message.dart

```diff
@@ -5,11 +5,25 @@
 /// An object sent from the Flutter Driver to a Flutter application to instruct
 /// the application to perform a task.
 abstract class Command {
+  Command({Duration timeout})
+      : this.timeout = timeout ?? const Duration(seconds: 5);
+
+  Command.deserialize(Map<String, String> json)
+      : timeout = new Duration(milliseconds: int.parse(json['timeout']));
+
+  /// The maximum amount of time to wait for the command to complete.
+  ///
+  /// Defaults to 5 seconds.
+  final Duration timeout;
+
   /// Identifies the type of the command object and of the handler.
   String get kind;
 
   /// Serializes this command to parameter name/value pairs.
-  Map<String, String> serialize();
+  Map<String, String> serialize() => {
+        'command': kind,
+        'timeout': '${timeout.inMilliseconds}',
+      };
```

Comment:
Hmm, I'd format this like this:

```dart
Map<String, String> serialize() => {
  'command': kind,
  'timeout': '${timeout.inMilliseconds}',
};
```

Also, the map literal needs to be typed according to Flutter's style guide.